chia/rpc/full_node_rpc_api.py

```diff
@@ -782,11 +783,13 @@ async def get_fee_estimate(self, request: Dict) -> Dict[str, Any]:
         if "cost" in request:
             cost = uint64(request["cost"])
 
-        target_times = request["target_times"]
+        target_times: List[int] = request["target_times"]
         estimator: FeeEstimatorInterface = self.service.mempool_manager.mempool.fee_estimator
+        target_times.sort()
         estimates = [
             estimator.estimate_fee_rate(time_offset_seconds=time).mojos_per_clvm_cost * cost for time in target_times
         ]
+        estimates = monotonically_decrease(estimates)
```

Comment:
If you understand how the fee estimator can end up with estimates that are not monotonically decreasing, this might be a good place to explain that in a comment, to future readers